frontend/src/features/permits/types/application.d.ts

```diff
@@ -59,9 +59,9 @@ export interface Commodities {
 }
 
 export interface TermOversizeApplication {
-  startDate: Dayjs;
+  startDate: Dayjs | any;
```

Comment:
Could be a good time to setup a meeting with the devs to discuss how we want to handle date formats and how we send them between FE/BE/DB